### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by Lykhoyda**
## Description
Update Readme file with the latest instructions

## Related Issue Or Context

Closes: #<issue>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in sygma-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

**Review by P1sar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,44 +1,56 @@
...
 - docker -> [install](https://docs.docker.com/engine/install/)
 - docker-compose -> [install](https://docs.docker.com/compose/install/)
 
-In order for your to bridge tokens from one network to another, you are going to need to clone [Sygma](https://github.com/ChainSafe/sygma). This project contains everything you need to run a bridge with two `evm` networks, and all the contracts deployed. Check the [README](https://github.com/ChainSafe/sygma/blob/main/README.md) and follow the instructions to install and have everything ready.
+### Sygma setup
 
+To bridge tokens from one network to another, you need to clone
+[Sygma](https://github.com/ChainSafe/sygma). This project contains everything you need to run a bridge with two `EVM`
+networks, and all the contracts deployed. Check the [README](https://github.com/ChainSafe/sygma/blob/main/README.md)
+and follow the instructions to install and have everything ready.
```
Comment (BeroBurny) [inline README.md:44]: think the current repo URL is https://github.com/sygmaprotocol/sygma-relayer

File: README.md [hunk 2]
```diff
@@ -1,44 +1,56 @@
...
 # SygmaUI
 
 ## Introduction
 
 **Sygma UI** is an OpenSource (under GNU Lesser General Public License v3.0) whitelabel application for developers
-to work with [Sygma](https://github.com/ChainSafe/sygma). UI consist of two part:
-BridgeUI is used to interact with [Bridge](https://github.com/chainsafe/sygma-solidity) smart contracts
-in order to send deposits.
-ExplorerUI is used to track and navigate every bridging that happens over specific Bridge smart contract.
+to work with [Sygma](https://github.com/ChainSafe/sygma).
```
Comment (BeroBurny) [inline README.md:12]: The URL point to a private repo
```suggestion
to work with [Sygma](https://github.com/sygmaprotocol).
```

**Review by BeroBurny — CHANGES_REQUESTED** [review]
Overall, it looks good, but some of the URLs seem wrong.

**Comment by Lykhoyda** [comment]
> Overall, it looks good, but some of the URLs seem wrong.

Thank you for the review! I updated the code

File: README.md [hunk 3]
```diff
@@ -71,83 +89,130 @@ Generic resourceId 0x00000000000000000000000000000000000000000000000000000000000
...
-  INDEXER__URL: "http://localhost:8000",
-  SYGMA: {
-    chains: [
+```json
+{
+  "UI": {
+    "transactionAutoUpdateInterval": 5000,
+    "nftTokenPage": true
+  },
+  "SYGMA": {
```
Comment (enemycnt) [inline README.md:128]: please add latest config here and further:
````
  {
    domainId: "1",
    networkId: 1337,
    name: "Local EVM 1",
    decimals: 18,
    bridgeAddress: "0x6CdE2Cd82a4F8B74693Ff5e194c19CA08c2d1c68",
    erc20HandlerAddress: "0x02091EefF969b33A5CE8A729DaE325879bf76f90",
    erc721HandlerAddress: "0xC2D334e2f27A9dB2Ed8C4561De86C1A00EBf6760",
    rpcUrl: "http://localhost:8545",
    tokens: [
      {
        type: "erc20",
        address: "0x78E5b9cEC9aEA29071f070C8cC561F692B3511A6",
        name: "ERC20LRTST",
        symbol: "ETHIcon",
        imageUri: "ETHIcon",
        decimals: 18,
        resourceId:
          "0x0000000000000000000000000000000000000000000000000000000000000300",
        feeSettings: {
          type: "basic",
          address: "0x8dA96a8C2b2d3e5ae7e668d0C94393aa8D5D3B94",
        },
      },
    ],
  },
  {
    domainId: "2",
    networkId: 1338,
    name: "Local EVM 2",
    decimals: 18,
    bridgeAddress: "0x6CdE2Cd82a4F8B74693Ff5e194c19CA08c2d1c68",
    erc20HandlerAddress: "0x02091EefF969b33A5CE8A729DaE325879bf76f90",
    erc721HandlerAddress: "0x481f97f9C82a971B3844a422936a4d3c4082bF84",
    rpcUrl: "http://localhost:8547",
    tokens: [
      {
        type: "erc20",
        address: "0x78E5b9cEC9aEA29071f070C8cC561F692B3511A6",
        name: "ERC20LRTST",
        symbol: "ETHIcon",
        imageUri: "ETHIcon",
        decimals: 18,
        resourceId:
          "0x0000000000000000000000000000000000000000000000000000000000000300",
        feeSettings: {
          type: "basic",
          address: "0x8dA96a8C2b2d3e5ae7e668d0C94393aa8D5D3B94",
        },
      },
    ],
  },

**Review by enemycnt — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Lykhoyda) [reply README.md:128]: The config you provided is from the actual dev environment, is it correct?
I think that as you follow the steps in the readme, in the previous section you run the sygma locally and get the environment variables:
```text
- Chain 1 -
Bridge: 0xd606A00c1A39dA53EA7Bb3Ab570BBE40b156EB66
ERC20: 0x75dF75bcdCa8eA2360c562b4aaDBAF3dfAf5b19b
ERC20 Handler: 0xb83065680e6AEc805774d8545516dF4e936F0dC0
ERC721: 0xb911DF90bCccd3D76a1d8f5fDcd32471e28Cc2c1
ERC721 Handler: 0x05C5AFACf64A6082D4933752FfB447AED63581b1
Generic Handler: 0x7573B1c6de00a73e98CDac5Cd2c4a252BdC87600
Asset Store: 0x3cA3808176Ad060Ad80c4e08F30d85973Ef1d99e
ERC20 resourceId: 0x0000000000000000000000000000000000000000000000000000000000000000
ERC721 resourceId 0x0000000000000000000000000000000000000000000000000000000000000200
Generic resourceId 0x0000000000000000000000000000000000000000000000000000000000000100
```
So in the current step, we put the values from the local config to 'sygma-runtime-config.json'. It's better to keep the reference of the test environment as you can compare where the data is coming from. A user might be confused about why he/she gets the variables in the previous step and then uses a different config.

Comment (enemycnt) [reply README.md:128]: Yes, as an example, makes sense.

**Review by enemycnt — APPROVED** [review]

**Pull request merged**